mandatory/3-function-output.js

```diff
@@ -3,16 +3,24 @@ function getRandomNumber() {
   return Math.random() * 10;
 }
 
+// This function will return a random number greater than or equal to 1 or less than 10. It will be a decimal. We don't need to pass an argument through this function.
```

Comment:
Issue: This is not quite correct, the number will be greater than or equal to 0 and less than 10
For more info see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random